modules/ding_adgangsplatformen/ding_adgangsplatformen.module

```diff
@@ -209,7 +216,16 @@ function ding_adgangsplatformen_logout($hooked = FALSE) {
   $token = $_SESSION['oauth2token'];
 
   // Generate logout request for the authorization service and send the request.
-  $logout_url = $config['urlLogout'] . '?access_token=' . $token . '&redirect_uri=' . $base_url;
+  $logout_url = url(
+    $config['urlLogout'],
+    array(
+      'external' => true,
+      'query' => array(
+        'singlelogout' => 'true',
```

Comment:
This parameter should be configurable to allow to opt-out of singe sign out.

Reply:
:+1: Done.